cypress/support/fragments/inventory/inventorySearchAndFilter.js

```diff
@@ -581,5 +582,7 @@ export default {
     cy.wait(1000);
     holdingsPermanentLocationAccordion.find(TextField()).click();
     cy.do(holdingsPermanentLocationAccordion.find(Checkbox(location)).click());
-  }
+  },
+
+  checkRowsCount:(expectedRowsCount) => cy.expect(instancesList.find(MultiColumnListRow({ index: expectedRowsCount + 1 })).absent())
```

Comment:
do you wanna check for absent or exists?

Reply:
Here I am checking that (expectedRowsCount + 1)th row does not exist to verify that we don't have more rows than expected. But now I think I have to add a check that (expectedRowsCount)th exists to make sure that we don't have less rows than expected. Will do it

Reply:
I found the issue, thank you. `index` here starts with 0. Updated this method to this and verified it is working: `cy.expect(instancesList.find(MultiColumnListRow({ index: expectedRowsCount - 1 })).exists());
    cy.expect(instancesList.find(MultiColumnListRow({ index: expectedRowsCount })).absent());`